barretenberg/cpp/pil/vm2/execution.pil

```diff
@@ -26,16 +26,38 @@ include "nullifier_check.pil";
 namespace execution;
 
 pol commit sel; // subtrace selector
+// Subtrace operation id
+pol commit subtrace_operation_id;
+
+// Subtrace Dispatch selectors
+pol commit alu_sel;
+pol commit bitwise_sel;
+pol commit to_radix_sel;
+pol commit ecc_add_sel;
+pol commit poseidon2_perm_sel;
```

Comment:
Do we use `sel` as suffix anywhere else?

Does polygon do that as well? 😆

Reply:
haha force of habit - i guess we implicitly apply the `sel` suffix to boolean columns that "turn on/off" subtraces,  they are just done at codegen with the namespace.

Happy to change how we name these